Tests/accounts-journey/unit-tests/AccountsJourneyObservabilityTests.swift

```diff
@@ -0,0 +1,131 @@
+import XCTest
+import Resolver
+import BackbaseObservability
+@testable import AccountsJourney
+
+final class AccountsJourneyObservabilityTests: XCTestCase {
+    
+    private var userActionSpy: TrackerSpy<UserActionEvent>?
```

Comment:
What is the a reason to keep references as alive for these? You can create them with let in each test case, since the closure will capture them anyway.

Reply:
Instead of having to repeat them for every tests, i though of having them as optional instead and only assigning them during the test.